### ISSUE TITLE
Bibtex citation

### ISSUE THREAD

**Issue by blueCat11** [djnavarro/rbook#23]
Your book has been (and still is) very useful to me in my studies and consequently, I want to cite it in my thesis and seminar papers, for which I use LaTeX and Bitex. Author, year and title fields are no problem. 
However, I am unsure what type this work is (online? misc? book?), in what field to put the version, and what the publisher is? Would this be lulu.com? Or just you as the author?

**Comment by djnavarro** [comment]
Yeah, academic citation doesn't quite capture this case well. I would be tempted to cite it as though it were a book. The ISBN for the book was issued by lulu.com so technically they were the publisher for versions 0.5 and earlier. Version 0.6 was never issued an ISBN because it started to seem ridiculous. I would be tempted to list https://learningstatisticswithr.com in place of the publisher (not strictly correct but it's as close as you can get), and treat the "version" as though it were the same as "edition". In the next edition/version I'm planning to assign a DOI to the book but (as always) I am a little behind schedule on this!

**Comment by blueCat11** [comment]
Thanks for the pointers!
